webapp/src/client/app/modules/web-client/rdp/web-client-rdp.component.ts

```diff
@@ -53,13 +55,16 @@ export class WebClientRdpComponent extends WebClientBaseComponent implements OnI
   @Output() sizeChange: EventEmitter<void> = new EventEmitter<void>();
 
   @ViewChild('sessionRdpContainer') sessionContainerElement: ElementRef;
-  @ViewChild('ironGuiElement') ironGuiElement: ElementRef;
+  @ViewChild('ironRemoteDesktopElementRdp') ironRemoteDesktopElementRdp: ElementRef;
```

Comment:
question: Is it necessary to name it ironRemoteDesktopElementRdp over ironRemoteDesktopElement (it’s part of a component called WebClient**Rdp**Component)